akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpHeaderParser.scala

```diff
@@ -444,16 +444,11 @@ private[http] object HttpHeaderParser {
 
   private val alwaysParsedHeaders = Set[String](
     "connection",
-    "content-encoding",
     "content-length",
     "content-type",
     "expect",
     "host",
-    "sec-websocket-key",
-    "sec-websocket-protocol",
-    "sec-websocket-version",
-    "transfer-encoding",
-    "upgrade"
+    "transfer-encoding"
```

Comment:
Um, wasn't all those headers added for Play just now in another PR

Reply:
Whoops, I messed something up there it seems